src/models/serw13/README.md

```diff
@@ -0,0 +1,64 @@
+# Serval WS (serw13)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Serval WS](./img/serw13.webp)
+
+The System76 Serval WS is a laptop with the following specifications:
+
+- CPU
+    - Supports Intel 13th Generation (Raptor Lake) CPUs
+        - [Core i5-13500HX](https://ark.intel.com/content/www/us/en/ark/products/232156/intel-core-i513500hx-processor-24m-cache-up-to-4-70-ghz.html)
+        - [Core i7-13700HX](https://ark.intel.com/content/www/us/en/ark/products/232166/intel-core-i713700hx-processor-30m-cache-up-to-5-00-ghz.html)
+        - [Core i9-13900HX](https://ark.intel.com/content/www/us/en/ark/products/232171/intel-core-i913900hx-processor-36m-cache-up-to-5-40-ghz.html)
+- Chipset
+    - Intel HM770
+- BIOS
+    - GigaDevice GD25B256EYIGR
+        - WSON-8 form factor
+    - Programmed with [System76 Open Firmware](https://github.com/system76/firmware-open)
+- EC
+    - ITE IT5570E
+    - Programmed with [System76 EC](https://github.com/system76/ec)
+- Graphics
+    - GPU options:
+        - NVIDIA GeForce RTX 4050
+        - NVIDIA GeForce RTX 4060
+        - NVIDIA GeForce RTX 4070
+    - eDP display: 15.6" 1920x1080@144Hz LCD
+        - LCD panel: BOE NV156FHM-NY8 (or equivalent)
+    - External video outputs:
+        - 1x HDMI
+        - 1x Mini DisplayPort 1.4
+        - 2x DisplayPort 1.4 over USB-C
+- Memory
+    - Up to 64GB (2x32GB) dual-channel DDR5 SO-DIMMs @ 5600 MHz
+- Networking
+    - 2.5-Gigabit Ethernet
+    - M.2 PCIe/CNVi WiFi/Bluetooth
+        - Intel Wi-Fi 6E AX211
+- Power
+    - 280W (20V, 14A) DC-in port
+        - DC-in port accepts rectangular (not barrel) connector
+    - Included AC adapter: Chicony A18-280P1A
+        - AC power cord type: IEC C13
+    - 80Wh 6-cell battery
+- Sound
+    - Internal speakers & microphone
+    - Combined headphone & microphone 3.5mm jack
+    - Combined microphone & S/PDIF 3.5mm jack
+    - HDMI, Mini DisplayPort, USB-C DisplayPort audio
+- Storage
+    - 2x PCIe NVMe Gen 4
+    - MicroSD card reader
+- USB
+    - 1x USB Type-C with Thunderbolt 4
+        - Supports USB-PD (charging) and DisplayPort over USB-C.
```

Comment:
How did we test this? USB-PD only goes up to 240W with the new current spec.

Reply:
I should probably add a note that it's only enabled when the machine's powered off.

Reply:
Addressed: 732cb7d82b32e56ea7b6c1384eed4bbde6426417